### ISSUE TITLE
[FEAT]: Info of audio track to "select audio track" list

### ISSUE THREAD

**Issue by Emporio1992** [nova-video-player/aos-AVP#615]
### Description

When Movie have multiple audiotracks, name of different audiotracks should content the info of those tracks rather than just name (track 1, track 2 etc...)

To get my point, see how Just player handles the track selection screen

**Comment by Pentaphon** [comment]
@courville duplicate of https://github.com/nova-video-player/aos-AVP/issues/556 I think?

Overall, both of them are asking for more info in the track selector.

**Comment by Emporio1992** [comment]
> @courville duplicate of #556 I think?
> 
> Overall, both of them are asking for more info in the track selector.

Yes, except #556 only talks about language. Language is only part of story. The format of the track is at least as important as language

**Comment by Pentaphon** [comment]
> Yes, except #556 only talks about language. Language is only part of story. The format of the track is at least as important as language

Yep, might as well describe every aspect like every player should.
